source/standards/optimise-frontend-perf.html.md.erb

```diff
@@ -27,6 +27,7 @@ For example:
 ||Investigate the impact of loading [@font-face][] assets on perceived performance |Investigate the CSS `font-display` property for managing common issues like [FOUT, FOIT and FOFT][]|
 |#Low|Set images and sprites correctly|Set images and sprites horizontally as it’s easier for browsers to parse|
 ||Reduce cookie size|Because every cookie is sent with each HTTP request, consider using a cookie-free domain for static assets [HTTP/1.1 only]. Enable HTTP/2 to enable [HPACK][] header compression or HTTP/3 for [QPACK][] |
+||Investigate using a Content Delivery Network (CDN) | A CDN will improve site performance by using a network of servers to deliver resources to users. The user will get delivered resourses from the server that is located nearest to the user.  A CDN is well-suited to handling heavy traffic and traffic spikes.
```

Comment:
Good addition and clearly written!

Just a small typo and a double-space to fix:

```suggestion
||Investigate using a Content Delivery Network (CDN) | A CDN will improve site performance by using a network of servers to deliver resources to users. The user will get delivered resources from the server that is located nearest to the user. A CDN is well-suited to handling heavy traffic and traffic spikes.
```

Reply:
Looks great!

Reply:
Good spot - thanks